receiver/receivercreator/README.md

```diff
@@ -1,10 +1,14 @@
 # Receiver Creator
 
-| Status                   |                       |
-|--------------------------|-----------------------|
-| Stability                | [beta]                |
-| Supported pipeline types | metrics               |
-| Distributions            | [contrib]             |
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [beta]: logs, metrics, traces   |
```

Comment:
is beta the right level of support for the other signals here?

Reply:
you know what, since we have to rebase, I'll make logs and traces alpha.